common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -52,6 +52,10 @@ div.problem {
 
   .inline {
     display: inline;
+
+    + p {
```

Comment:
This is an adjacency selector. In particular, it selects p tags that immediately follow any element that has class "inline" (they're used a lot in the edX css presently).

See: http://www.w3schools.com/cssref/css_selectors.asp